### ISSUE TITLE
oc observe is broken

### ISSUE THREAD

**Issue by kargakis** [openshift/origin#13779]
```
$ oc observe svc --loglevel=8
I0415 14:11:52.501042   30596 loader.go:354] Config loaded from file /var/run/kubernetes/admin.kubeconfig
I0415 14:11:52.502928   30596 round_trippers.go:296] GET https://localhost:6443/api
I0415 14:11:52.502943   30596 round_trippers.go:303] Request Headers:
I0415 14:11:52.502952   30596 round_trippers.go:306]     Accept: application/json, */*
I0415 14:11:52.502961   30596 round_trippers.go:306]     User-Agent: oc/v1.5.0 (linux/amd64) openshift/c3eade6
I0415 14:11:52.531594   30596 round_trippers.go:321] Response Status: 200 OK in 28 milliseconds
I0415 14:11:52.531615   30596 round_trippers.go:324] Response Headers:
I0415 14:11:52.531621   30596 round_trippers.go:327]     Content-Type: application/json
I0415 14:11:52.531635   30596 round_trippers.go:327]     Content-Length: 135
I0415 14:11:52.531642   30596 round_trippers.go:327]     Date: Sat, 15 Apr 2017 12:11:52 GMT
I0415 14:11:52.531668   30596 request.go:905] Response Body: {"kind":"APIVersions","versions":["v1"],"serverAddressByClientCIDRs":[{"clientCIDR":"0.0.0.0/0","serverAddress":"192.168.0.13:6443"}]}
I0415 14:11:52.531811   30596 round_trippers.go:296] GET https://localhost:6443/apis
I0415 14:11:52.531820   30596 round_trippers.go:303] Request Headers:
I0415 14:11:52.531825   30596 round_trippers.go:306]     Accept: application/json, */*
I0415 14:11:52.531830   30596 round_trippers.go:306]     User-Agent: oc/v1.5.0 (linux/amd64) openshift/c3eade6
I0415 14:11:52.532735   30596 round_trippers.go:321] Response Status: 200 OK in 0 milliseconds
I0415 14:11:52.532748   30596 round_trippers.go:324] Response Headers:
I0415 14:11:52.532753   30596 round_trippers.go:327]     Content-Type: text/plain; charset=utf-8
I0415 14:11:52.532758   30596 round_trippers.go:327]     Content-Length: 2841
I0415 14:11:52.532762   30596 round_trippers.go:327]     Date: Sat, 15 Apr 2017 12:11:52 GMT
I0415 14:11:52.532800   30596 request.go:905] Response Body: {"kind":"APIGroupList","apiVersion":"v1","groups":[{"name":"apiregistration.k8s.io","versions":[{"groupVersion":"apiregistration.k8s.io/v1alpha1","version":"v1alpha1"}],"preferredVersion":{"groupVersion":"apiregistration.k8s.io/v1alpha1","version":"v1alpha1"},"serverAddressByClientCIDRs":null},{"name":"authentication.k8s.io","versions":[{"groupVersion":"authentication.k8s.io/v1","version":"v1"},{"groupVersion":"authentication.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"authentication.k8s.io/v1","version":"v1"},"serverAddressByClientCIDRs":null},{"name":"authorization.k8s.io","versions":[{"groupVersion":"authorization.k8s.io/v1","version":"v1"},{"groupVersion":"authorization.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"authorization.k8s.io/v1","version":"v1"},"serverAddressByClientCIDRs":null},{"name":"autoscaling","versions":[{"groupVersion":"autoscaling/v1","version":"v1"}],"preferredVersion":{"groupVersion":"autoscaling/v1","version":"v1"},"serverAddressByClientCIDRs":null},{"name":"batch","versions":[{"groupVersion":"batch/v1","version":"v1"}],"preferredVersion":{"groupVersion":"batch/v1","version":"v1"},"serverAddressByClientCIDRs":null},{"name":"storage.k8s.io","versions":[{"groupVersion":"storage.k8s.io/v1","version":"v1"},{"groupVersion":"storage.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"storage.k8s.io/v1","version":"v1"},"serverAddressByClientCIDRs":null},{"name":"rbac.authorization.k8s.io","versions":[{"groupVersion":"rbac.authorization.k8s.io/v1alpha1","version":"v1alpha1"},{"groupVersion":"rbac.authorization.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"rbac.authorization.k8s.io/v1alpha1","version":"v1alpha1"},"serverAddressByClientCIDRs":null},{"name":"settings.k8s.io","versions":[{"groupVersion":"settings.k8s.io/v1alpha1","version":"v1alpha1"}],"preferredVersion":{"groupVersion":"settings.k8s.io/v1alpha1","version":"v1alpha1"},"serverAddressByClientCIDRs":null},{"name":"certificates.k8s.io","versions":[{"groupVersion":"certificates.k8s.io/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"certificates.k8s.io/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":null},{"name":"extensions","versions":[{"groupVersion":"extensions/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"extensions/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":null},{"name":"policy","versions":[{"groupVersion":"policy/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"policy/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":null},{"name":"apps","versions":[{"groupVersion":"apps/v1beta1","version":"v1beta1"}],"preferredVersion":{"groupVersion":"apps/v1beta1","version":"v1beta1"},"serverAddressByClientCIDRs":null}]}
F0415 14:11:52.532850   30596 helpers.go:121] error: serializer for text/plain; charset=utf-8 doesn't exist
```

On a side note, we just keep latest in DockerHub, can we start cutting tags for it? Commit hashes for example?

@smarterclayton

**Comment by kargakis** [comment]
Hm, this is against a Kubernetes cluster from master. Works fine with openshift/origin:v1.5.0-alpha.3.

**Comment by smarterclayton** [comment]
That content type is wrong.  Looks like we regressed serving content type for the external apis. @sttts

**Comment by kargakis** [comment]
This is fixed with oc 1.6
```
$ oc version
oc v3.6.0-alpha.1+bea9588-382
kubernetes v1.6.1+5115d708d7
features: Basic-Auth

Server https://localhost:6443/
kubernetes v1.7.0-alpha.2.591+fe174044c456c3-dirty
```
